packages/vertica-nodejs/test/unit/connection/inbound-parser-tests.js

```diff
@@ -9,6 +9,7 @@ require('./test-helper')
 const BufferList = require('../../buffer-list')
 var Connection = require('../../../lib/connection')
 var buffers = require('../../test-buffers')
+const { tls_mode } = require('../../../lib/defaults')
```

Comment:
Is this require necessary?

Reply:
Good catch, we aren't using the default tls_mode which is now 'prefer' anywhere in the test. Since we want to use tls_node as 'disable', this is not longer needed, so I removed it.